lib/guardian.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
-
```

Comment:
Could you add this line back in, so that we don't have history in the file where nothing actually changed

Reply:
Okay